weave-js/src/components/Panel2/ChildPanelExportReport/customSelectComponents.tsx

```diff
@@ -43,25 +51,53 @@ export const customEntitySelectComps = {
 };
 
 export const customReportSelectComps = {
-  MenuList: (props: MenuListProps<ReportOption, false>) => (
+  SingleValue: ({
+    children,
+    ...props
+  }: SingleValueProps<ReportOption, false, GroupedReportOption>) => (
+    <selectComponents.SingleValue
+      {...props}
+      className="flex items-center gap-8">
+      <Icon name="add-new" width={18} height={18} />
+      {children}
+    </selectComponents.SingleValue>
+  ),
+  MenuList: (
+    props: MenuListProps<ReportOption, false, GroupedReportOption>
+  ) => (
     <selectComponents.MenuList
       {...props}
       className="max-h-[calc(100vh-34rem)]"
     />
   ),
-  Option: ({children, ...props}: OptionProps<ReportOption, false>) => {
+  GroupHeading: (
+    groupHeadingProps: GroupHeadingProps<
+      ReportOption,
+      false,
+      GroupedReportOption
+    >
+  ) => {
+    const isFirstGroup =
+      groupHeadingProps.selectProps.options.findIndex(
+        option => option === groupHeadingProps.data
+      ) === 0;
+    return (
+      <selectComponents.GroupHeading {...groupHeadingProps} className="">
+        {!isFirstGroup && <div className="my-8 h-1 bg-moon-250"></div>}
```

Comment:
**suggestion (non-blocking, accessibility):**  add `role="separator"` or use `hr` instead of `div`

Reply:
TIL about `hr`. I had copied this over from `Select` and it was using `div`, but didn't touch it in this PR.